.github/rawWorkflows/gh-ci-parameterized-flow.txt

```diff
@@ -11,6 +12,9 @@
     needs: $Dependency
     if: $Conditional
     timeout-minutes: $TimeOut
+    concurrency:
+     group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}-jdk${{ matrix.jdk }}-$FlowName
+     cancel-in-progress: true
```

Comment:
For the primary repo, can we only cancel ongoing runs of the current pull request when it gets updated. For non-PRs, we should let them run

Reply:
This is what I was thinking of 
```
cancel-in-progress:  ${{ github.repository != 'linkedin/venice' || github.event_name == 'pull_request' }}
```